Dungeoneer/src/com/interrupt/dungeoneer/entities/infos/LevelInfo.java

```diff
@@ -0,0 +1,129 @@
+package com.interrupt.dungeoneer.entities.infos;
+
+import com.badlogic.gdx.graphics.Color;
+import com.interrupt.dungeoneer.annotations.EditorProperty;
+import com.interrupt.dungeoneer.entities.Entity;
+import com.interrupt.dungeoneer.game.Level;
+
+import java.util.Objects;
+
+public class LevelInfo extends Entity {
+    /** Name of level. */
+    @EditorProperty
+    public static String levelName = "";
+
+    /** Ambient color. */
+    @EditorProperty
+    public static Color ambientColor = new Color(Color.BLACK);
+
+    /** Camera far draw distance. */
+    @EditorProperty
+    public static float viewDistance = 15f;
+
+    /** Starting distance of fog. */
+    @EditorProperty
+    public static float fogStart;
```

Comment:
Default value?

Reply:
No. The values will get set on init().

Reply:
Just wondering because all the other properties have default values.

Reply:
Ooh. Good point. Let me think on this. Either way, I should be consistent. :)